Qr code component

Daniel4lm opened the pull request:
## Subject

This pull request contains the solution to the QR code component challenge on Frontend Mentor as a part of my Optimum BH Internship. The request was to implement QR code component by using HTML/CSS/Tailwind web technologies and get it looking as close as possible to the design provided in internship material.

### The solution on this branch contains the following:

- [x] Html page containing the corresponding DOM structure, including the Optimum header with anchor tags with urls(whose presense was requested by script tests).
- [x] Tailwind configuration file containing the extended theme atributes, like colors, sizes, breakpoints, etc.
- [x] Challenge starter files, incl. images and style guide
- [x] Solution design images(mobile & desktop layout)
- [x] CSS style file with tailwind directives and custom classes

Daniel4lm committed:
Add basic styling and configuration

- added challenge starter files
- removed old chromedriver ver from package.json
- added additional tailwind styling

Daniel4lm committed:
Add QR card styling

- Updated qr_code index.html with style and content
- tailwind conf extended theme
- Added group styling for qr card inside style.css

Daniel4lm committed:
Update main index.html with link to current challenge

Daniel4lm committed:
Removed chromedriver from package.json

Daniel4lm committed:
Fix layout breakpoints style

Daniel4lm committed:
Fix link name in root index.html

Daniel4lm committed:
Fix inside package.json file

Daniel4lm committed:
Fix inside package.json file

Daniel4lm committed:
Merge branch 'optimumBA:qr_code_component' into qr_code_component

Daniel4lm committed:
Remove unused settings

Daniel4lm committed:
Add missing goggle font import

Daniel4lm committed:
Replace and move some design files

- Add readme.md and template files
- Include final design images

Daniel4lm committed:
Merge branch 'optimumBA:qr_code_component' into qr_code_component

Daniel4lm committed:
Remove unused parts of code

Daniel4lm committed:
Modify readme.md

Daniel4lm committed:
Minor css improvements

Daniel4lm committed:
Merge branch 'optimumBA:qr_code_component' into qr_code_component

almirsarajcic commented on challenges/qr_code_component/style.css:
```diff
@@ -1,3 +1,10 @@
 @tailwind utilities;
+
+@layer components {
+  .qr-card {
```
Nice touch! 👏

almirsarajcic commented on challenges/index.html:
```diff
@@ -51,11 +53,14 @@
       </div>
 
       <div class="mt-3">
+        <p>QR code component -</p>
```
This file should only be changed in the `main` branch.

Daniel4lm replied:
Forgot to revert changes back inside this html file.

almirsarajcic reviewed (changes requested):
Great work! 👍

Just a small issue: make the credits visible in mobile variant as well.

Daniel4lm committed:
Changes requested

- Remove redundant code inside main index.html file
- Delete README.md & style-guide.md files
- Make credits visible in mobile layout(hidden before during tests)

Daniel4lm committed:
Fix source code link
